extras/build/index.js

```diff
@@ -1,10 +1,19 @@
-require('./webfonts.js')
-require('./animate')
-
-require('./material-icons.js')
-require('./mdi-v5.js')
-require('./fontawesome-v5.js')
-require('./ionicons-v5.js')
-require('./eva-icons.js')
-require('./themify.js')
-require('./line-awesome.js')
+const parallel = require('os').cpus().length > 1
+const runJob = parallel ? require('child_process').fork : require
+const { join } = require('path')
+
+runJob(join(__dirname, './webfonts.js'))
+runJob(join(__dirname, './animate.js'))
+
+// replaced by material-icons2
+// runJob(join(__dirname, './material-icons.js'))
+
+runJob(join(__dirname, './mdi-v5.js'))
+runJob(join(__dirname, './fontawesome-v5.js'))
+runJob(join(__dirname, './ionicons-v5.js'))
+runJob(join(__dirname, './eva-icons.js'))
+runJob(join(__dirname, './themify.js'))
+runJob(join(__dirname, './line-awesome.js'))
+
+// this one takes the longest
+runJob(join(__dirname, './material-icons2.js'))
```

Comment:
Is original `material-icons.js` still used? If not, can we replace it with the new version instead of creating a duplicate?

Reply:
I agree, also the related dependency `material-design-icons` should be removed too.

Reply:
material-icons2 renamed to material-icons

Reply:
As for material-design-icons, I will look into removing it. However, Razvan likes a license file and that's the only known existing on for the material svg icons. Suggestions? @IlCallo @yusufkandemir

Reply:
Don't really have much experience on licenses, but can't we just copy that one over?

Reply:
@IICallo From where?

Reply:
Since the license files have already been copied over and are in Git, I will just leave them there. `material-design-icons` package removed.